src/replaceCustomValues.ts

```diff
@@ -221,22 +317,15 @@ export function getElementSubmissionValue({
         continue
       }
 
-      const result = getElementSubmissionValue({
+      const result = getElementSubmissionValueById({
         elementId,
-        propertyName,
         formElements: element.elements,
```

Comment:
When iterating over pages, we need to ensure that the elements available are all elements across all pages/sections.

Reply:
As discussed, I was confusing this logic with replacing values in context.